boosters/node-beginner/tests/config/setup.js

```diff
@@ -1,3 +1,10 @@
 const db = require("../../models");
+UniqueConstraintError = require("sequelize").UniqueConstraintError;
```

Comment:
No longer need this require as we are capturing all errors thrown.

Reply:
Thanks